land_claims.civmap.json

```diff
@@ -241,5 +241,13 @@
+{"id":"ac713034-c3b1-4654-bd22-c9370ec06fa6","collectionId":"civmap:collection/user","polygon":[[[7586,-7109],[7925,-7106],[7931,-6861],[7638,-6887],[7548,-7025],[7581,-7106]]],"name":"Union (Buckeye)"},
+{"id":"b56e0ff0-3ad0-4df8-ab44-f0e769d38acf","collectionId":"civmap:collection/user","polygon":[[[3119,12727],[3054,12683],[3062,12599],[3061,12569],[3136,12494],[3202,12460],[3215,12368],[3267,12343],[3462,12354],[3678,12433],[3727,12512],[3371,12760]]],"name":"Fight Club Island"},
```

Comment:
generally it would be cool if the name would be the first field, like with the existing entries